### ISSUE TITLE
Add newly modified IEEE 13 node model for FastDERMS

### ISSUE THREAD

**Issue by poorva1209** [GRIDAPPSD/GOSS-GridAPPS-D#1646]
- with batteries and pv
- this will add to blazegraph:fastderms docker image

**Comment by shpoudel** [comment]
I don't think this can be closed. There were a few utility-scale PVs added (I think by Venkat) to the model when I looked into the new blazegraph that Tonya provided. I think it needs further modification  (for example, adding batteries). Folks from LBL will be looking more here. So, basically, we need a consensus from NREL and LBL before I can work on this.

**Comment by poorva1209** [comment]
Ok, assigning it to you Shiva for now. Let's follow up with NREL and LBL and let Tonya know when it is ready for a release.
